pkg/apis/core/validation/validation_test.go

```diff
@@ -24193,6 +24193,64 @@ func TestValidateSleepAction(t *testing.T) {
 	}
 }
 
+// TODO: merge these test to TestValidatePodSpec after AllowRelaxedDNSSearchValidation feature graduates to Beta
+func TestValidatePodDNSConfigWithRelaxedSearchDomain(t *testing.T) {
+	testCases := []struct {
+		name           string
+		expectError    bool
+		featureEnabled bool
+		dnsConfig      *core.PodDNSConfig
+	}{
+		{
+			name:           "beings with underscore, contains underscore, featuregate enabled",
```

Comment:
s/beings/begins ?

Reply:
Yup, thanks!
Fixed in 1b7aa4558f0346b0cc69a12493324b4b61feeba2